src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/DraftAdjudicationDto.kt

```diff
@@ -23,6 +24,8 @@ data class DraftAdjudicationDto(
   val startedByUserId: String? = null,
   @Schema(description = "Is classified as a youth offender")
   val isYouthOffender: Boolean? = null,
+  @Schema(description = "Damages related to incident")
+  val damages: List<DamageDto>? = null
 )
```

Comment:
slightly surprised that this isn't an empty list, typically easier to deal with then a null list (same as the other list)

Reply:
Its come from offences and other sections.  As the adjudication is built up over a number of steps, i think the view is not to initialise things until they are actually set.  Slightly different in this area, as not selecting any damages is acceptable, but we decided that this should only be set to an empty list at the point the draft is locked into a reported adjudication (ie user confirms that the data is correct - ie missing)